.gitignore

```diff
@@ -10,9 +10,9 @@ npm-debug.log
 /node_modules/.bin
 /docs/public/
 /docs/.cache/
+/docs/.DS_Store
```

Comment:
I believe this can be `.DS_Store` at the top-level and it _should_ work for all the nested folders